apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -30,7 +30,7 @@
     "FACILITIES_DESC": "Manage facilities and resources.",
     "ADMIN_DESC": "Manage attributes, users and do diagnostics.",
     "PROFILE_DESC": "View information about you and your organizations and groups.",
-    "MY_PROFILE": "My profile"
+    "MY_ACCOUNT": "My account"
```

Comment:
I'm not sure if we want to rename this page to "My account". I get that you wanted to avoid duplicated profiles but we might want to remove the old profile completely. I would suggest keeping the labels the way they were.